libs/react-spa/shared/src/components/problems/Problem.tsx

```diff
@@ -114,7 +114,7 @@ export const Problem = ({
           return (
             <ThirdPartyServiceError
               organizationSlug={organizationSlug}
-              size={size}
+              size={size ?? 'small'}
```

Comment:
This is changing the default sizing from `large` to `small` for everyone using the shared component. I´m not sure that is what we want. What is the goal here? When do you need it small?

Reply:
According to the designs when the error is from a specific service provider, we want to display a small warning. 
![image](https://github.com/island-is/island.is/assets/9449413/d818d0de-9eb8-4b3e-9505-67a85f7ce210)
However, when the error doesn't come from a specific source, we want the large box.
![image](https://github.com/island-is/island.is/assets/9449413/c48f7db5-2d82-459c-9d0d-d093b4733155)

Somehow overlooked how far-reaching this change would be.